templates/database/database-ss.yaml

```diff
@@ -63,7 +63,7 @@ spec:
         image: {{ .Values.database.internal.image.repository }}:{{ .Values.database.internal.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         command: ["/bin/sh"]
-        args: ["-c", "chmod -R 700 /var/lib/postgresql/data/pgdata || true"]
+        args: ["-c", "chmod -R 700 /var/lib/postgresql/data/pgdata || true && chmod -R 700 /var/lib/postgresql/data/pgdata/pg* || true"]
```

Comment:
maybe it should be:
```
args: ["-c", "chmod -R 700 /var/lib/postgresql/data/pgdata || chmod -R 700 /var/lib/postgresql/data/pgdata/pg* || true"]
```
rather than having `&&`